test/global_test_setup.clj

```diff
@@ -1,7 +1,12 @@
 (ns global-test-setup
   "This namespace's contents will be automatically loaded by `lein test` (or any test runner)."
   (:require
-   [refactor-nrepl.inlined-deps.toolsnamespace.v1v1v0.clojure.tools.namespace.repl :refer [set-refresh-dirs]]))
+   [clojure.tools.namespace.repl :refer [set-refresh-dirs]]))
+
+(when (find-ns
```

Comment:
find-ns can return nil if nothing required the ns before. i.e. it's not a reliable way to detect whether a ns exists

I'd recommend to `(try (require ...))` directly instead

Reply:
Good idea, thanks! 👍